config/capi/capi.yml

```diff
@@ -137,3 +137,22 @@ metadata:
 type: Opaque
 stringData:
   password: #@ data.values.capi.database.encryption_key
+
+#! overlays for scaling capi
+#@overlay/match by=overlay.subset({"kind":"Deployment", "metadata":{"name":"cf-api-server"}})
+---
+#@overlay/match-child-defaults missing_ok=True
+spec:
+  replicas: #@ data.values.capi.scale.cf_api_server.replicas
+  template:
+    spec:
+      containers:
+      - #@overlay/match by=overlay.subset({"name":"cf-api-server"})
+        resources:
```

Comment:
If we want to allow QoS guaranteed I think we'll have to expose every single container in the deployment.

Reply:
Hi @loewenstein. Thanks for the feedback. We are still working on implementing all of the required overlays to make use of the scaling configuration we added to `config/values/00-values.yml`. We wanted to get this submitted quickly to get more eyes on the proposed user interface, but recognise that the work is a long way from being complete and ready to merge. I have switched the PR to a draft in the meantime to reflect this.

Reply:
Hi @davewalter,

I realized that after commenting, when I had seen the proposed values. 

Might be worth checking with @cppforlife if we could traverse the data values structure and create overlays for each container without all the repetition.